src/app/pages/new-task/new-task.page.spec.ts

```diff
@@ -1,22 +1,22 @@
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
I think this were never maintained. We can get better feeling if we remove tests that never work so there will be no confusion. No strong opinion, but I think we forgot to remove them :D

Reply:
Yeah I was quite tempted to get rid of the tests :p

Reply:
Go with your instincts.